tasks/rpmbuild-test

```diff
@@ -30,25 +31,26 @@ uuidgen > file.lock
 cp file.lock uuid.saved
 while true; do
      # Check if lock exists on remote server
-     while [[ $(rsync --dry-run -avz file.lock fedora-atomic@artifacts.ci.centos.org::fedora-atomic/${RSYNC_BRANCH}/repo/lockdir) != *"file.lock"* ]]; do
+     while [[ $(rsync --ignore-existing --dry-run -avz file.lock fedora-atomic@artifacts.ci.centos.org::fedora-atomic/${RSYNC_BRANCH}/repo/lockdir) != *"file.lock"* ]]; do
           sleep 60
      done
+     cp uuid.saved file.lock
```

Comment:
That isn't my point here.  If you push your file.lock out, then pull it back and it doesn't match because you got someone else's file.lock, you now do not have your own uuid in file.lock for the next iteration of the loop.  Or ever, for that matter